packages/component/src/Attachment/Text/private/MarkdownTextContent.tsx

```diff
@@ -41,6 +41,10 @@ function isCitationURL(url: string): boolean {
   return onErrorResumeNext(() => new URL(url))?.protocol === 'cite:';
 }
 
+function getEntryName(entry: Entry) {
+  return entry.claim?.appearance?.usageInfo?.name;
```

Comment:
Should this account for other possible names?

```suggestion
  return entry.claim?.name ?? entry.claim?.appearance?.usageInfo?.name;
```

Reply:
Despite the name of that field and this function, I think `entry.claim?.name` and `entry.claim?.appearance?.usageInfo?.name` are different things. I'll see if I can at put some explanatory comments on this.